kerl

```diff
@@ -171,7 +171,7 @@ usage()
 {
     echo "kerl: build and install Erlang/OTP"
     echo "usage: $0 <command> [options ...]"
-    printf "\n  <command>       Command to be executed\n\n"
```

Comment:
Oh, doh

I swear I tested that, but either way it's not even really related to this PR so I'll roll that out.